### ISSUE TITLE
Visualization: Image(s)

### ISSUE THREAD

**Issue by brunobergher** [metabase/metabase#27674]
It would be nice to be able to simply display an image in a card.
Its URL could come from a single record, with us pulling a semantically appropriate column by default (which could be overriden).
And in the case of multiple records, we could show a carrousel or some sort.

Example use cases:
- Logos in customer dashboards
- Photos in product dashboards
- Screenshots/frames in data pipelines
- etc

**How important is this feature to you?**
Not necessary, but pretty cool, especially in operational contexts.

**Comment by paoliniluis** [comment]
Just tried in dashboard 863 on our stats instance and it's possible, type "website" in the filter

**Comment by brunobergher** [comment]
Yup, it works for dashboard parameters, this would be for records themselves.
You can hack it together for dashboards where there's a parameter which can be interpolated into a URL, but that's super specific.

**Comment by dragonsahead** [comment]
Metabase now supports custom visualizations in v62. Feel free to create the new visualizations and import those in your Metabase instance. You can check these examples:
https://github.com/metabase/custom-viz-thumbs
https://github.com/metabase/custom-viz-calendar-heatmap
